### PR TITLE
(bug): adjust active navbar menu when selected

### PR DESCRIPTION
**Pull request by fandok**
image

### Summary by CodeRabbit

**Release Notes:**

- New Feature: Added smooth scrolling behavior to the webpage for enhanced user experience.
- Refactor: Updated navigation bar to dynamically render menu items and highlight the active item based on URL hash, improving usability.
- Style: Improved code readability by formatting component props and splitting content variables into multiple lines.
- Chore: Removed unused imports and refactored components to use hooks directly from 'react' instead of 'React'.
- New Feature: Introduced environment variables for Clerk service keys, preparing for integration with Clerk authentication service.

### CODE REVIEW

File: src/app/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -44,30 +57,26 @@ export default function App({ featureRef }) {
...
+                className="text-white text-lg"
+                color="foreground"
+                href={isHome ? "#" : `#${menu.toLowerCase()}`}
+              >
+                {menu}
+              </Link>
+            </NavbarItem>
+          );
+        })}
       </NavbarContent>
```
[the remark this answers is not in the document]
Comment (fandok) [reply src/app/components/Navbar/Navbar.jsx:80]: apply this

File: src/app/components/Hero/Hero.jsx [hunk 2]
```diff
@@ -45,7 +47,7 @@ export default function Hero() {
           </div>
         </div>
         <div className="sm:w-2/5 sm:mt-[63px]">
-          <img src={logo.src} />
+          <img alt="logo" src={logo.src} />
         </div>
```
[the remark this answers is not in the document]
Comment (fandok) [reply src/app/components/Hero/Hero.jsx:51]: apply this also

File: src/app/components/Navbar/Navbar.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (fandok) [reply src/app/components/Navbar/Navbar.jsx:80]: @coderabbitai can you apply this?

File: src/app/components/Hero/Hero.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (fandok) [reply src/app/components/Hero/Hero.jsx:51]: @coderabbitai can you help also apply this

**Review by Swpn0neel — APPROVED** [review]

**Review by Swpn0neel — APPROVED** [review]

**Review by Swpn0neel — APPROVED** [review]

**Comment by Arindam200** [comment]
Hey @fandok ! It's working for the PC version. 

Can you please add the same feature to the mobile one?

**Comment by fandok** [comment]
> Hey @fandok ! It's working for the PC version.
> 
> Can you please add the same feature to the mobile one?

Sure! Let me work on it today

**Review by Swpn0neel — APPROVED** [review]

**Review by Swpn0neel — APPROVED** [review]

**Comment by fandok** [comment]
@Arindam200 already solved it for mobile version. let me know if it is okay for you or not

**Review by Arindam200 — APPROVED** [review]
LGTM

**Comment by Arindam200** [comment]
Thanks a lot for Contributing!

**Pull request merged**